src/components/auth/Login.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { Formik } from 'formik';
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  Container,
+  TextField,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+import './styles.css';
```

Comment:
Can you rename the `styles.css` to `Login.css`? Since the CSS belongs to the component.